pkg/packagekit/autoupdates.jsx

```diff
@@ -296,7 +296,7 @@
                             <FormSelect id="auto-update-day"
                                          isDisabled={pending}
                                          value={day == "" ? "everyday" : day}
-                                         onChange={d => setDay(d == "everyday" ? "" : d) }>
+                                         onChange={(_, d) => setDay(d == "everyday" ? "" : d) }>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)